mindsdb/migrations/versions/2024-07-18_459a4cd24933_merge_divergent_branches.py

```diff
@@ -0,0 +1,23 @@
+"""merge divergent branches
+
+Revision ID: 459a4cd24933
+Revises: cbedc4968d5d, bfc6f44f5bc9
+Create Date: 2024-07-18 23:38:29.998117
+
+"""
+import mindsdb.interfaces.storage.db  # noqa
+
+
+# revision identifiers, used by Alembic.
+revision = '459a4cd24933'
+down_revision = ('cbedc4968d5d', 'bfc6f44f5bc9')
```

Comment:
I think it should be fixed by https://github.com/mindsdb/mindsdb/pull/9518/files

Reply:
Ok will remove